.gitignore

```diff
@@ -4,6 +4,7 @@ https
 *.js.map
 dist
 .secret_config.json
```

Comment:
can we get rid of this now? Should we delete `.secret_config.json`?

Reply:
I think keeping it makes sense as anyone who already has the repo pulled down would end up committing the .secret_config.json on their next push